package.json

```
@@ -4,15 +4,25 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node server.js"
+    "compile": "tsc",
+    "start": "npm run compile && node ./dist/index.js",
+    "dev": "nodemon --exec tsx src/index.ts"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@apollo/server": "5.0.0",
     "express": "5.1.0",
     "graphql": "16.11.0",
     "graphql-http": "1.22.4",
     "ruru": "2.0.0-0.13"
   },
-  "type": "module"
+  "type": "module",
+  "devDependencies": {
+    "@types/node": "24.3.0",
+    "nodemon": "^3.1.10",
+    "ts-node": "^10.9.2",
+    "tsx": "^4.20.4",
```

Poderia usar as versões exatas aqui. Se colocar a flag `--save-exact` no npm install, ele já faz assim direto 
```suggestion
    "nodemon": "3.1.10",
    "ts-node": "10.9.2",
    "tsx": "4.20.4",
```